recipe/meta.yaml

```diff
@@ -121,20 +125,18 @@ outputs:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
         - aiohttp >=3.7
-        - aiosignal
         - aiohttp-cors
         - aioredis <2
         - colorful
-        - frozenlist
-        # gpustat-0.6.0 has a dependency which does not exist on Windows;
-        # skip it there until gpustat is fixed as it is optional
+        # gpustat has a dependency which does not exist on Windows;
+        # skip it there until gpustat is fixed as it is optional.
         - gpustat  # [not win]
+        - opencensus
+        - prometheus_client >=0.7.1,<0.14.0
         - jsonschema
         - requests
-        - opencensus
-        - prometheus_client >=0.7.1
-        - py-spy >=0.2.0
         - smart_open
+        - py-spy >=0.2.0
```

Comment:
pydantic is also listed: https://github.com/ray-project/ray/blob/ray-2.0.1/python/setup.py#L232

Reply:
@cbouss Good catch! I removed `aioredis` and `redis-py`; these are indeed no longer used anywhere in the code base.
`pydantic` needs to be added to `ray-default` instead of `ray-core`: `ray-dashboard` needs `ray-default` instead of `core`. I verified these changes with the recipe on `conda-forge` too, although in the end it doesn't really matter since we don't build `ray-dashboard` anyway.

Reply:
Cool!